src/facade/reply_builder_test.cc

```diff
@@ -745,6 +745,23 @@ TEST_F(RedisReplyBuilderTest, SendStringArrayAsSet) {
       << "SendStringArrayAsSet Resp3 Failed.";
 }
 
+TEST_F(RedisReplyBuilderTest, SendNumSubArray) {
```

Comment:
No need for this test after you remove the function :)

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2282/commits/0abca50381ec31ee7332cc95617af1ec9dad1eae